### DISCUSSION TITLE
Error when installing All of Create server pack

### DISCUSSION THREAD

**Discussion by Simper0082** [itzg/docker-minecraft-server#3068] [Q&A]
Hello,

I keep getting the below error whenever I try to install the server pack found here https://www.curseforge.com/minecraft/modpacks/aoc/files/5656862
-- Head --                                                                                                              Thread: main                                                                                                            Suspected Mods: NONE                                                                                                    Stacktrace:                                                                                                                     at net.minecraftforge.logging.CrashReportExtender.lambda$dumpModLoadingCrashReport$7(CrashReportExtender.java:5>-- MOD jerintegration --                                                                                                Details:                                                                                                                        Mod File: /data/mods/JER-Integration-4.5.0.jar                                                                          Failure message: Mod jerintegration requires jeresources 0.14.1.160 or above                                                    Currently, jeresources is not installed                                                                         Mod Version: 4.5.0                                                                                                      Mod Issue URL: NOT PROVIDED                                                                                             Exception message: MISSING EXCEPTION MESSAGE            

Is there a reason this would not be installing with the installer? 

yaml I am running is below

services:
  mc:
    image: itzg/minecraft-server:java17
    tty: true
    stdin_open: true
    ports:
      - "25565:25565"
    environment:
      EULA: "TRUE"
      TYPE: "FORGE"
      CF_API_KEY: **************
      CF_SERVER_MOD: https://www.curseforge.com/minecraft/modpacks/aoc/files/5656862
    volumes:
      # attach the relative directory 'data' to the container's /data path
      - ./data:/data
     
Thanks for the hand

**Comment by Simper0082** [comment]
Resolution reached

services:
  mc:
    image: itzg/minecraft-server:java17
    tty: true
    stdin_open: true
    ports:
      - "25565:25565"
    environment:
      EULA: "TRUE"
      TYPE: AUTO_CURSEFORGE
      CF_API_KEY: ************************
      CF_EXCLUDE_MODS: jer-integration
      CF_PAGE_URL: https://www.curseforge.com/minecraft/modpacks/aoc/files/5656821
      MEMORY: 10G
    volumes:
      # attach the relative directory 'data' to the container's /data path
      - ./data:/data

**Reply by mrpinkcat** [reply]
Ty 🙏
